lib/secrets-manager/secrets-manager.ts

```diff
@@ -0,0 +1,13 @@
+import { KeyValueManager } from 'lib/key-value-manager/key-value-manager.abstract';
+import { IKeyValueManager } from 'types/key-value-manager';
+
+// TODO: create new release on github -  with notes on env-vars not being secured anymore
```

Comment:
@maorb-dev that's not true, they will continue to be the same - exposed in process.env, but still secured. Let's discuss before publishing something like that